src/librustdoc/html/static/rustdoc.css

```diff
@@ -390,17 +390,13 @@ nav.sub {
 	cursor: pointer;
 }
 
+.docblock-short {
+	overflow-wrap: anywhere;
+}
 .docblock-short p {
 	display: inline;
 }
 
-.docblock-short.nowrap {
```

Comment:
If we're no longer adding styling for this, can we remove the `nowrap` class wherever it's used?

Reply:
I removed it because we have no `nowrap` class generated anywhere, not the other way around. ;)